lib/kv-redis.js

```diff
@@ -42,10 +63,30 @@ function RedisKeyValueConnector(settings, dataSource) {
 
 util.inherits(RedisKeyValueConnector, Connector);
 
+/**
+ * Returns "PONG". Used to verify connectivity to the data source.
+ *
+ * @callback {Function} callback
+ * @param {Error} err Error object.
+ * @param {String} message The "PONG" message string.
+ *
+ * @header RedisKeyValueConnector.ping(cb)
```

Comment:
I am not sure whether it's a good idea to rename `RedisKeyValueConnector.prototype.ping` to `RedisKeyValueConnector.ping`. Is this something we are doing in other connectors too?

I don't mind either way, just trying to better understand what is our convention.

(This comment applies to all other places where we drop `.prototype` from the `@header`.)